### PR TITLE
feat(FX-3507): Artworks text styling to Palette V3

### PR DESCRIPTION
**Pull request by dimatretyak**
The type of this PR is: **Feature**

This PR resolves [FX-3507]

### Description
* Search Results - Artworks Tab (Pill) / Artwork text description is not following Palette v3 styles
* Artwork grids across the app (except the Home Feed) is not rendering in the updated font styles.

### Acceptance criteria
* Lot Number — UNICA REGULAR CAPS [XS] Black100
* Artist Name — Unica Regular [xs] Black100
* Artwork Title — Unica Regular Italic [xs] 
* Black60Artwork Year — Unica Regular [xs] Black60
* Partner — Unica Regular [xs] Black 60
* Artwork Price — Unice Medium [xs] Black100

### Demo
|Before|After|
|---|---|
||before|
||before-1|
||before-2|

### PR Checklist (tick all before merging)

- [x] I have included screenshots or videos to illustrate my changes, or I have not changed anything that impacts the UI.
- [x] I have tested my changes on **iOS** and **Android**.
- [x] I have added tests/stories for my changes, or my changes don't require testing/stories, or I have included a link to a separate Jira ticket covering the tests.
- [x] I have added a feature flag, or my changes don't require a feature flag. ([How do I add one?](https://github.com/artsy/eigen/blob/master/docs/developing_a_feature.md))
- [x] I have documented any follow-up work that this PR will require, or it does not require any.
- [x] I have added an app state migration, or my changes do not require one. ([What are migrations?](https://github.com/artsy/eigen/blob/master/docs/adding_state_migrations.md))
- [x] I have added a changelog entry below or my changes do not require one.

### To the reviewers 👀

- [x] I would like at least one of the reviewers to run this PR on the simulator or device.

<details><summary>Changelog updates</summary>

#### Cross-platform user-facing changes

- Artworks text styling to Palette V3 - dimatretyak

#### iOS user-facing changes

-

#### Android user-facing changes

-

#### Dev changes

-

</details>

[FX-3507]: https://artsyproduct.atlassian.net/browse/FX-3507?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/lib/Components/ArtworkGrids/ArtworkGridItem.tsx [hunk 1]
```diff
@@ -151,28 +151,37 @@ export const Artwork: React.FC<ArtworkProps> = ({
...
         <Box mt={1}>
           {!!showLotLabel && !!artwork.saleArtwork?.lotLabel && (
-            <Text color="black60" numberOfLines={1} {...lotLabelTextStyle}>
+            <Text variant="xs" color="black100" numberOfLines={1} caps {...lotLabelTextStyle}>
               Lot {artwork.saleArtwork.lotLabel}
             </Text>
           )}
           {!!artwork.artistNames && (
-            <Text lineHeight="20" weight="medium" numberOfLines={1} {...artistNamesTextStyle}>
+            <Text lineHeight="18" weight="regular" variant="xs" numberOfLines={1} {...artistNamesTextStyle}>
```
Comment (gkartalis) [inline src/lib/Components/ArtworkGrids/ArtworkGridItem.tsx:159]: **question:** do we need to explicitly declare here as in the lotLabel above `color="black100"` or does it fallback to this without writing it?

File: src/lib/Scenes/MyCollection/Screens/ArtworkList/MyCollectionArtworkListItem.tsx [hunk 2]
```diff
@@ -77,12 +77,16 @@ const MyCollectionArtworkListItem: React.FC<MyCollectionArtworkListItemProps> =
       <View>
         {renderImage()}
         <Box maxWidth={width} mt={1} style={{ flex: 1 }}>
-          <Sans size="4">{artistNames}</Sans>
+          <Text lineHeight="18" weight="regular" variant="xs" numberOfLines={1}>
```
Comment (gkartalis) [inline src/lib/Scenes/MyCollection/Screens/ArtworkList/MyCollectionArtworkListItem.tsx:80]: **question:** do we need to explicitly declare here as in the `ArtworkGridItem` above color="black100" or does it fallback to this without explicitly declaring it?

**Review by gkartalis — APPROVED** [review]
Looking good, just wrote two questions for clarification

File: src/lib/Components/ArtworkGrids/ArtworkGridItem.tsx (hunk 1 above)
Comment (dimatretyak) [reply src/lib/Components/ArtworkGrids/ArtworkGridItem.tsx:159]: We can do without explicitly specifying `color="black 100"`

**Pull request merged**